public/index.html

```diff
@@ -1,32 +1,34 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>Fluent Playground</title>
 
     <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-77033033-23"></script>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=UA-77033033-23"
```

Comment:
Do we actually need this?

Reply:
Probably not. I'll drop it in a separate cleanup PR.